Packs/Base/ReleaseNotes/1_15_9.md

```diff
@@ -0,0 +1,6 @@
+#### Scripts
+##### SanePdfReports
+- Update the Docker image to: *demisto/sane-pdf-reports:1.0.0.24891*
+- 6.5 includes the following updates:
```

Comment:
I would drop the bug fixes notes, and just keep the docker image update 
other than that looks good

Reply:
done, @Itay4 please approve 🚀